cpu/nrf51/periph/pwm.c

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_nrf51
+ * @ingroup     drivers_periph_pwm
+ * @{
+ *
+ * @file
+ * @brief       Low-level PWM driver implementation
+*
+ * @note        This driver initializes a duty cycle of '1'
+ *
+ * @author      Semjon Kerner <semjon.kerner@fu-berlin.de>
+ * @}
+ */
+
+#include <stdint.h>
+#include <string.h>
+#include <math.h>
+
+#include "periph/gpio.h"
+#include "periph/pwm.h"
+#include "vectors_cortexm.h"
+
+#define ENABLE_DEBUG                (1)
+#include "debug.h"
+#define DEBUG_ASSERT_VERBOSE        (1U)
+
+#define NRF51_PWM_PRESCALER_MIN     (0U)
+#define NRF51_PWM_PRESCALER_MAX     (10U)
+#define NRF51_PWM_PPI_CHANNELS      (((1U) << PWM_PPI_A) | ((1U) << PWM_PPI_B))
+#ifndef NRF51_PWM_PERCENT_VAL
+    #define NRF51_PWM_PERCENT_VAL   (1U)
+#endif
+
+inline uint32_t apply_prescaler(uint8_t prescaler)
```

Comment:
use `static`.

And as I recently learned: no need to declare this `inline`, as the compiler will take care of this anyway.